src/components/AddRollupButtonYellowstone.jsx

```diff
@@ -4,16 +4,16 @@ export default function AddRollupButton() {
   const addNetwork = () => {
     const params = [
       {
-        chainId: '0x907',
-        chainName: 'Chronicle Vesuvius - Lit Protocol Testnet',
+        chainId: '175188',
```

Comment:
Are we sure this can be an int? Did you testing adding it to MetaMask?

Reply:
Yeah manually added it using an int and not hex, works 👍  tested with the faucet and got tokens to the chain